plugin/trino-sqlserver/src/test/java/io/trino/plugin/sqlserver/TestingSqlServer.java

```diff
@@ -126,6 +141,20 @@ private static void setUpDatabase(SqlExecutor executor, String databaseName, BiC
         databaseSetUp.accept(executor, databaseName);
     }
 
+    private static void waitCollationChange(MSSQLServerContainer<?> container)
+    {
+        // Creating objects before finishing system change leads to collation conflict error
+        // https://github.com/microsoft/mssql-docker/issues/724
+        for (int i = 0; i < 60; i++) {
+            if (container.getLogs().contains("The default collation was successfully changed")) {
```

Comment:
Seems like LogMessageWaitStrategy (in addition to existing one) would be simpler/useful?

Reply:
If I remember correctly, `JdbcDatabaseContainer` doesn't allow multiple wait strategies. I will take a look again.